README.md

```diff
@@ -246,17 +257,23 @@ Before continuing, check the following:
 
 To deploy:
 
-1. Go to [repository actions](/../../actions/workflows/web_app.yml) and click on _Run workflow_ and then the green button to run it.
+1. Navigate to your repository on Github and select the _Actions_ tab.
+2. Select _Build and deploy Python app to Azure Web App_ from the _Workflows_ sidebar.
+3. Select _Run workflow_ and select the green button inside the pop-up to run the workflow.
 
 **Deploying can take a couple of minutes**. Make sure you stream the logs in the Azure Cloud Shell to check the progress:
 
 ```
 az webapp log tail --name $AZURE_WEBAPP_NAME --resource-group $AZURE_RESOURCE_GROUP
 ```
 
+4. Once deployment is complete, visit your website at a URL like `http://demo-fastapi-97709018.azurewebsites.net/`,
+where the random number is your unique random number. You can find the website URL in the Azure portal or in the deployment logs if you forgot the number.
+5. 🎉 Celebrate a successful deployment! You now have a URL that you can share with classmates, friends, and family.
+
 ### Destroy resources when complete
 
-After deploying, make sure you cleanup your resources by destroying the resource group. You can do it in the Azure Cloud Shell by referencing the group name you created initially (`demo-fastapi` in the examples):
+You likely don't want to keep this particular website running forever in the cloud, so you should cleanup your Azure resources by destroying the resource group. You can do it in the Azure Cloud Shell by referencing the group name you created initially (`demo-fastapi` in the examples):
```

Comment:
I always find it odd how quick our tutorials are to tell us to clean stuff up. It's like hey, I like what I deployed!! Why so quick to destroy it? So I put a little preface here. Rewords welcome.